udata/templates/home.html

```diff
@@ -92,7 +92,7 @@ <h1 class="page-header">
                                 <h3 class="page-header">{{ _('Used datasets') }}</h3>
                             </div>
 
-                            {% for dataset in reuse.datasets[:4] %}
+                            {% for dataset in (reuse.datasets|rejectattr('deleted')|list)[:4] %}
```

Comment:
To me we should put as few logic as possible within templates and here we're chaining a bit too much filters for my taste. Why don't we store that in a variable at the view level?